test/command_callback/test_llc_command_callback.vader

```diff
@@ -0,0 +1,24 @@
+Before:
+  Save g:ale_llvm_llc_executable
+
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+
+  runtime ale_linters/llvm/llc.vim
+
+After:
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+  Restore
+
+Execute(llc command is customizable):
+  let cmd = ale_linters#llvm#llc#GetCommand(bufnr(''))[:3]
+  AssertEqual 'llc ', cmd
```

Comment:
When testing the executable names, use `ale#Escape` on them here. This will make the tests pass in both Unix and Windows environments. (Windows escaping is very different, and may not add quotes at all in some cases.)